src/app-layout/__tests__/drawers.test.tsx

```diff
@@ -1,27 +1,23 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
-/* eslint simple-import-sort/imports: 0 */
 import React from 'react';
-import { act, render, fireEvent } from '@testing-library/react';
+import { act, fireEvent, render } from '@testing-library/react';
+
 import { KeyCode } from '@cloudscape-design/test-utils-core/utils.js';
-import createWrapper from '../../../lib/components/test-utils/dom';
 
+import AppLayout, { AppLayoutProps } from '../../../lib/components/app-layout';
+import createWrapper from '../../../lib/components/test-utils/dom';
 import {
   describeEachAppLayout,
-  renderComponent,
-  testDrawer,
+  findActiveDrawerLandmark,
   manyDrawers,
   manyDrawersWithBadges,
-  findActiveDrawerLandmark,
+  renderComponent,
+  testDrawer,
 } from './utils';
 
-import AppLayout, { AppLayoutProps } from '../../../lib/components/app-layout';
-import visualRefreshStyles from '../../../lib/components/app-layout/visual-refresh/styles.selectors.js';
 import toolbarTriggerButtonStyles from '../../../lib/components/app-layout/visual-refresh-toolbar/toolbar/trigger-button/styles.css.js';
-
-jest.mock('../../../lib/components/internal/hooks/use-mobile', () => ({
-  useMobile: jest.fn().mockReturnValue(true),
-}));
```

Comment:
Here and in many places below – removed direct mock applications, because `describeEachAppLayout` helper takes care of setting up the correct environment